yt/visualization/volume_rendering/off_axis_projection.py

```diff
@@ -233,7 +235,8 @@ def off_axis_projection(data_source, center, normal_vector,
                     width.to('code_length').d,
                     chunk[weight].to(wounits),
                     weight_buff,
-                    normal_vector)
+                    normal_vector,
+                    north)
 
             buf /= weight_buff
```

Comment:
I know this isn't your line, but I've just realised that this line could return NaN's, particularly with the scatter approach. I actually added some utility functions to the pixelization routines for doing this.

You should be able to do 
```python
from yt.utilities.lib.pixelization_routines import normalization_2d_utility

...

normalization_2d_utility(buf, weight_buff)
```
These utility functions just take two 2D memoryview's and check if the denominator is not zero and then do the division.

Reply:
Which line are you talking about?  Line 239/242?  And you want me to include the added code just prior to that line?

Reply:
so the current line `buf /= weight_buff` (L241) which I know isn't yours can lead to NaN's. But if you include `from yt.utilities.lib.pixelization_routines import normalization_2d_utility` at the top of the file then that line can be replaced with `normalization_2d_utility(buf, weight_buff)` which is much more NaN safe